jobs/create-version-branch.js

```diff
@@ -86,7 +86,7 @@ module.exports = async function (
     return
   }
 
-  if (repository.private) {
+  if (repository.private && !env.IS_ENTERPRISE) {
```

Comment:
🎨 in https://github.com/greenkeeperio/greenkeeper/pull/602/commits/a13f03b905faaa9eb1a6e6112bf1ba3619c0fb8a#diff-137e495c3f0b01ee061147fede39cbacR67 you put the `IS_ENTERPRISE` check to the front. We should agree on a uniform way, which one doesn’t matter.

Reply:
Will do. I'd opt for `private` first, as this allows for some early exits and most of the special behaviour only affects private repos anyway.

Reply:
I like that. In addition, that’s the main code logic, and Enterprise is a secondary consideration, so it works well at the end.

Reply:
b771be9 done